### PR TITLE
fix: can not delete br in table by delete key (close #405)

### LINKED ISSUE #405: Unexpected column table when press delete key in the table of Wysiwyg (2MD)
## Version
v1.3.1

## Test Environment
Chrome

## Current Behavior

1. In Wysiwyg, press delete key in the table.
2. Unexpected columns appears.

## Expected Behavior

Should not appear unexpected columns.

### PR DESCRIPTION
**Pull request by sohee-lee7**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has a description of the breaking change

### Description

fix #405 

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/wwTableManager.js [hunk 1]
```diff
@@ -405,26 +405,57 @@ class WwTableManager {
...
+   * Return whether delete br in the br
+   * @param {Range} range Range object
+   * @returns {boolean}
+   */
+  _isBRDeleting(range) {
+    const currentNode = this._getCurrentNodeInCell(range);
+    const nextSibling = currentNode && currentNode.nextSibling;
+
+    return !!(domUtils.getNodeName(currentNode) === 'BR' &&
+              nextSibling && domUtils.getNodeName(nextSibling) === 'BR');
```
Comment (dongwoo-kim) [inline src/js/wwTableManager.js]: 사소한건데..  이 코드는 `!!`가 왜 붙었는지를 잠깐 고민하게 만드네요. 
전체가 아니라 `nextSibling` 앞에만 `!!` 붙여주는 게 더 명확하지 않을까요?

File: src/js/wwTableManager.js [hunk 2]
```diff
@@ -405,26 +405,57 @@ class WwTableManager {
...
 
     return isCellDeleting || isEndOfText || isLastCellOfRow;
   }
+
+  /**
+   * Return whether delete br in the br
+   * @param {Range} range Range object
+   * @returns {boolean}
+   */
+  _isBRDeleting(range) {
```
Comment (dongwoo-kim) [inline src/js/wwTableManager.js]: 사소하지만.. `isDeletingBR`이 낫지 않나요? Deleting은 능동이라.. ㅎㅎ

File: src/js/wwTableManager.js (hunk 1 above)
Comment (sohee-lee7) [reply src/js/wwTableManager.js]: 아하 그렇네요! 반영하겠습니다

File: src/js/wwTableManager.js (hunk 2 above)
Comment (sohee-lee7) [reply src/js/wwTableManager.js]: 아 ㅋㅋ 넵! 기존의 isNonTextDeleting도 함께 변경하겠습니다.

**Pull request merged**